make/autoconf/flags-ldflags.m4

```diff
@@ -70,7 +70,8 @@ AC_DEFUN([FLAGS_SETUP_LDFLAGS_HELPER],
       fi
     fi
 
-    BASIC_LDFLAGS_JVM_ONLY="-Wl,-O1"
+    # Moved optimization flags to JvmFeatures.gmk, was -Wl,-O1
```

Comment:
Please delete. This type of comment has no context when not seen in this PR. Nobody knows something else used to be here.

Reply:
Alright, will do